### ISSUE TITLE
Possible new issue: Make small initial contributions to documentation easier

### ISSUE THREAD

**Issue by paulehoffman** [bagder/github-feedback#3]
The current requirement to either fork (and understand forking) or edit files in the GItHub GUI makes it difficult to get newbies who have a small correction to get started. There should be a way for someone to make a near-trivial PR (such as adding an example, or even just correcting a grammar mistake) in a project's documentation without the normal overhead. The project manager can then encourage the new contributor to step up to the normal process for larger contributions.

**Comment by baseely** [comment]
I agree with you @paulehoffman. Do you have an imagination for how this could work ? May be the GitHub GUI needs some tweaks/ UX for to ease the process for new contributors while maintaining the same CONTRIBUTING guides and documentation requirements (Formats, ...).
